decidim-proposals/app/assets/javascripts/decidim/proposals/admin/proposals.es6

```diff
@@ -24,7 +24,7 @@ $(document).ready(function () {
     }
   }
 
-  let hideBulkActionsButton = function(force = false) {
+  window.hideBulkActionsButton = function(force = false) {
```

Comment:
Mh, I'm not sure about these changes, but JS is not my forte :( Maybe @leio10?

Reply:
This function is used when bulk changing the scope or category, once the form is submited the response is a `js` that triggers `hideBulkActionsButton` and others.

If it was not attached to the window it didn't work.

https://github.com/decidim/decidim/pull/5759/files/73cbbfc5cba5a0533b5ed52fb4c4f1410090dd91#diff-92de21341488f07c3a12733cabb4b7fcR23-R26

https://github.com/decidim/decidim/pull/5759/files/73cbbfc5cba5a0533b5ed52fb4c4f1410090dd91#diff-b81adb75152bd426cdeb72d3a6bcfe4bL22-R24

Reply:
👍